connection.go

```diff
@@ -89,10 +89,30 @@ func processSetStatement(ctx context.Context, c *fireboltConnection, query strin
 		return false, nil
 	}
 
-	_, err = c.client.Query(ctx, c.engineUrl, c.databaseName, "SELECT 1", map[string]string{setKey: setValue})
+	if setKey == "database" {
+		return true, fmt.Errorf("`SET database` query is not allowed. Please use `USE database` syntax")
+	}
```

Comment:
Should we let the server take care of the error messaging? If at some point they'll allow SET database we'll have to remove this.

Reply:
Makes sense, will remove this check